.github/BOTMETA.yml

```diff
@@ -906,11 +906,19 @@ files:
 ###############################
 # plugins/filter
   $plugins/filter/:
+    support: community
+  $plugins/filter/__init__.py:
```

Comment:
doesn't this need to be just the dir?

Reply:
`$plugins/filter/` is `community, unless specified otherwise